aspnetcore/performance/caching/middleware.md

```diff
@@ -13,44 +14,31 @@ uid: performance/caching/middleware
+This article explains how to configure Response Caching Middleware in an ASP.NET Core app. The middleware determines when responses are cacheable, stores responses, and serves responses from cache. For an introduction to HTTP caching and the `ResponseCache` attribute, see [Response Caching](xref:performance/caching/response).
 
 ## Package
-To include the middleware in a project, add a reference to the [`Microsoft.AspNetCore.ResponseCaching`](https://www.nuget.org/packages/Microsoft.AspNetCore.ResponseCaching/) package or use the [`Microsoft.AspNetCore.All`](https://www.nuget.org/packages/Microsoft.AspNetCore.All/) package.
 
-## Configuration
-In `ConfigureServices`, add the middleware to the service collection.
-
-# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+To include the middleware in a project, add a reference to the [`Microsoft.AspNetCore.ResponseCaching`](https://www.nuget.org/packages/Microsoft.AspNetCore.ResponseCaching/) package or use the [`Microsoft.AspNetCore.All`](https://www.nuget.org/packages/Microsoft.AspNetCore.All/) package (ASP.NET Core 2.0 or later).
 
-[!code-csharp[Main](middleware/samples/2.x/Program.cs?name=snippet1&highlight=4)]
-
-# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+## Configuration
 
-[!code-csharp[Main](middleware/samples/1.x/Startup.cs?name=snippet1&highlight=3)]
+In `ConfigureServices`, add the middleware to the service collection.
 
----
+[!code-csharp[Main](middleware/sample/Startup.cs?name=snippet1&highlight=3)]
 
 Configure the app to use the middleware with the `UseResponseCaching` extension method, which adds the middleware to the request processing pipeline. The sample app adds a [`Cache-Control`](https://tools.ietf.org/html/rfc7234#section-5.2) header to the response that caches cacheable responses for up to 10 seconds. The sample sends a [`Vary`](https://tools.ietf.org/html/rfc7231#section-7.1.4) header to configure the middleware to serve a cached response only if the [`Accept-Encoding`](https://tools.ietf.org/html/rfc7231#section-5.3.4) header of subsequent requests matches that of the original request.
 
-# [ASP.NET Core 2.x](#tab/aspnetcore2x)
-
-[!code-csharp[Main](middleware/samples/2.x/Program.cs?name=snippet1&highlight=8)]
-
-# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+[!code-csharp[Main](middleware/sample/Startup.cs?name=snippet2&highlight=3,7-12)]
 
-[!code-csharp[Main](middleware/samples/1.x/Startup.cs?name=snippet2&highlight=3)]
-
----
-
-The Response Caching Middleware only caches 200 (OK) server responses. Any other responses, including [error pages](xref:fundamentals/error-handling), are ignored by the middleware.
+Response Caching Middleware only caches 200 (OK) server responses. Any other responses, including [error pages](xref:fundamentals/error-handling), are ignored by the middleware.
```

Comment:
This first sentence could be confusing to some readers. As is, it sounds like 200 represents the number of responses cached rather than an HTTP status code.